src/components/SignInForm.tsx

```diff
@@ -12,7 +12,7 @@ export default function SignInForm() {
             height={100}
             priority
             src="/images/logo.svg"
-            style={{height: "100px"}}
+            style={{ height: "100px" }}
```

Comment:
Can we move this styling to a `className` and use tailwind to define it? 

It should be something like `h-80` (default sizing), or we can also define a custom value if it's really needed `h-[100px]`